package.json

```diff
@@ -59,11 +60,17 @@
   "engines": {
     "node": ">=10.15.1"
   },
-  "files": [ "dist" ],
+  "files": [
+    "dist"
```

Comment:
You'll need to add `lib` here if you want it included in the npm package I believe.